spec/01-unit/19-hybrid/03-compat_spec.lua

```diff
@@ -10,6 +10,12 @@ local function reset_fields()
 end
 
 describe("kong.clustering.compat", function()
+  lazy_teardown(function()
+    if _G.kong.db then
+      _G.kong.db:schema_reset()
```

Comment:
Should we move this logic to `teardown()` in line 393? Because only this test suite is using `truncate()`.

Reply:
I'm not sure if the tests after line 393 depend on these `truncate()`s, so I put it at the outermost level to ensure the schema is reset after all tests are executed.